gateway/src/main/resources/application.yml

```diff
@@ -23,7 +23,13 @@ spring:
     banner-mode: off
     web-application-type: reactive
   application.name: gateway-service
-  thymeleaf.cache: false
+  thymeleaf:
+    cache: false
+    check-template: true
+    check-template-location: true
+    enabled: true
+    prefix: classpath:/templates/
+    suffix: .html
```

Comment:
I am surprised that thymeleaf templates are relocated into the classpath, on the MEL project we located the templates/ subdirectory into the georchestra datadir.
Anyway, I think it is also fine this way, as long as we can still customize the configuration so that an external directory is used.

Reply:
Here thymeleaf should be configured due to redirection to gateway by Apache proxy, we need to set suffix (which is most important). In gateway templates is under /ressources which is referenced by classpath.